### PR TITLE
EPAS 15 - Added content as per DB-1985

### PR DESCRIPTION
**Pull request by nidhibhammar**
https://enterprisedb.atlassian.net/browse/DB-1985

### CODE REVIEW

File: product_docs/docs/epas/15/epas_guide/03_database_administration/01_configuration_parameters/01_setting_new_parameters.mdx [hunk 1]
```diff
@@ -87,3 +87,81 @@ application_name=psql
...
 "local   all             all               md5"
 ```
+
+### Obfuscating the LDAP password
+
+You can use LDAP authentication to authenticate database users. For more information , see [LDAP authentication](https://www.postgresql.org/docs/15/auth-ldap.html).
+
+The `ldapbindpasswd` is stored in plaintext in the pg_hba.conf file. It is possible to store an encrypted `ldapbindpasswd` by writing an extension to decrypt. This extension should contain a decryption function with the C signature. You can add this extension to the `shared_preload_libraries` parameter in postgresql.conf file. You can restart your server to load the changes in this parameter.
+
+```ini
```
Comment (drothery-edb) [inline product_docs/docs/epas/15/epas_guide/03_database_administration/01_configuration_parameters/01_setting_new_parameters.mdx:97]: I think we want to make the example into three steps: 1) creating the function 2) adding it to the conf file 3) Restart your server. Move this code block to the second step.

File: product_docs/docs/epas/15/epas_guide/03_database_administration/01_configuration_parameters/01_setting_new_parameters.mdx [hunk 2]
```diff
@@ -87,3 +87,81 @@ application_name=psql
...
+shared_preload_libraries = '$libdir/ldap_password_func'
+```
+
+This example shows the code of extension `ldap_password_func.c` with the function `rot13_passphrase` to decrypt the password:
+
```
Comment (drothery-edb) [inline product_docs/docs/epas/15/epas_guide/03_database_administration/01_configuration_parameters/01_setting_new_parameters.mdx:102]: 1. Create the function.

File: product_docs/docs/epas/15/epas_guide/03_database_administration/01_configuration_parameters/01_setting_new_parameters.mdx [hunk 3]
```diff
@@ -87,3 +87,81 @@ application_name=psql
...
+```c
+/*-------------------------------------------------------------------------
+ *
+ * ldap_password_func.c
+ *
+ * Loadable PostgreSQL module to mutate the ldapbindpasswd. This
```
Comment (drothery-edb) [inline product_docs/docs/epas/15/epas_guide/03_database_administration/01_configuration_parameters/01_setting_new_parameters.mdx:108]: If you are going to keep this comment, rewrite it for clarity using the same terminology used above.

File: product_docs/docs/epas/15/epas_guide/03_database_administration/01_configuration_parameters/01_setting_new_parameters.mdx [hunk 4]
```diff
@@ -87,3 +87,81 @@ application_name=psql
...
+
+		if ((c >= 'a' && c <= 'm') || (c >= 'A' && c <= 'M'))
+			*p = c + 13;
+		else if ((c >= 'n' && c <= 'z') || (c >= 'N' && c <= 'Z'))
+			*p = c - 13;
+	}
+
+	return new_pw;
+}
+```
```
Comment (drothery-edb) [inline product_docs/docs/epas/15/epas_guide/03_database_administration/01_configuration_parameters/01_setting_new_parameters.mdx:166]: 1. Add the extension to the `shared_preload_libraries` parameter in the postgresql.conf file.
1. Restart your server.

File: product_docs/docs/epas/15/epas_guide/03_database_administration/01_configuration_parameters/01_setting_new_parameters.mdx (hunk 1 above)
Comment (nidhibhammar) [reply product_docs/docs/epas/15/epas_guide/03_database_administration/01_configuration_parameters/01_setting_new_parameters.mdx:97]: I am not sure if this example can be done in these 3 steps or not.  As per the comments on the JIRA, we its not possible to have 100% working example here.

**Review by drothery-edb — APPROVED** [review]

**Pull request merged**